docs/pages/workflow/ios-simulator.md

```diff
@@ -8,25 +8,31 @@ import { Terminal } from '~/ui/components/Snippet';
 
 It's often convenient to develop your app directly on your computer rather than having to physically interact with an iPhone and iPad and load your app over the network, which may be slow under some conditions such as if you need to use a tunnel connection because LAN isn't possible on your network.
 
-This guide explains how you can install the iOS simulator on your Mac and use it for developing your app. It is not possible to install the iOS Simulator on any operating system except macOS; if you want to develop an app for iOS from a Windows machine then you will need to use a physical iOS device.
+This guide explains how you can install the iOS simulator on your Mac and use it for developing your app. It is not possible to install the iOS Simulator on any operating system except macOS. If you want to develop an app for iOS from a Windows machine then you will need to use a physical iOS device.
 
 ## Step 1: Install Xcode
 
-This step is very easy but it takes a while. Open up the Mac App Store, search for Xcode, and hit install (or update if you have it already). If you're not able to update because your operating system is out of date, we recommend updating your operating system to the latest version and then updating Xcode. You may run into issues further down the line if your Xcode version is out of date, for example you may not be able to submit your app to the App Store.
+This step is very easy but it takes a while. Open up the Mac App Store, search for [Xcode](https://apps.apple.com/us/app/xcode/id497799835), and hit install (or update if you have it already). 
+If you're not able to update because your operating system is out of date, we recommend updating your operating system to the latest version and then updating Xcode. 
+You may run into issues further down the line if your Xcode version is out of date, for example you may not be able to submit your app to the App Store.
```

Comment:
```suggestion
Open up the Mac App Store, search for [Xcode](https://apps.apple.com/us/app/xcode/id497799835), and hit install (or update if you have it already).
If you're unable to update, it is because your operating system might be out of date. We recommend updating your operating system to the latest version and then updating Xcode. 
You may run into issues further down the line if your Xcode version is out of date. For example, you may not be able to submit your app to the App Store.
```